### PR TITLE
Add a condition on site settings controller

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: app/site/site_settings/site_settings_controller.js [hunk 1]
```diff
@@ -80,7 +80,14 @@ function SiteSettingsController($scope, $rootScope, $state, $mdToast, API, Modal
         AuthService.updateAuth($scope.user);
         $rootScope.$broadcast('gonevisDash.SiteSettingsController:remove');
         $mdToast.showSimple("Site deleted");
-        ModalsService.open("sites", "SiteController");
+
+        // If user have no sites, redirect to "create a new site".
```
Comment (amirm3hdi) [inline app/site/site_settings/site_settings_controller.js]: No commenting needed here

File: app/site/site_settings/site_settings_controller.js [hunk 2]
```diff
@@ -80,7 +80,14 @@ function SiteSettingsController($scope, $rootScope, $state, $mdToast, API, Modal
...
+        if ($scope.user.sites.length == 0) {
+          $state.go('dash.site-new');
+        } else {
+          # Or let user choose a site.
```
Comment (amirm3hdi) [inline app/site/site_settings/site_settings_controller.js]: Remove comment

Comment (Alir3z4) [inline app/site/site_settings/site_settings_controller.js]: without this tiny comment here, it's not clear why we're making this line there.

File: app/site/site_settings/site_settings_controller.js (hunk 1 above)
Comment (Alir3z4) [inline app/site/site_settings/site_settings_controller.js]: Just removed this line

**Pull request merged**